src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments/Infrastructure/OuterApi/OuterApiService.cs

```diff
@@ -106,5 +106,10 @@ public async Task<GetCohortDetailsResponse> GetCohortDetails(long providerId, lo
         {
             return await _outerApiClient.Get<GetCohortDetailsResponse>(new GetCohortDetailsRequest(providerId, cohortId));
         }
+
+        public async Task<GetDraftApprenticeshipResponse> GetDraftApprenticeship(long providerId, long cohortId, long draftApprenticeshipId)
+        {
+            return await _outerApiClient.Get<GetDraftApprenticeshipResponse>(new GetDraftApprenticeshipRequest(providerId, cohortId, draftApprenticeshipId));
+        }
```

Comment:
I like putting this call here, but I don't think weu are ever calling it. Your code is calling the Outer API directly in the mapper

Reply:
yep, agreed, fixed